!Command
positional: []
named: []
command:
- pypi_downloads
parent:
subcommands: []
help_flag:
usage_flag:
version_flag:
help_text: "Traceback (most recent call last):\n  File \"/tmp/tmpego0l9bo/bin/pypi_downloads\"\
  , line 9, in <module>\n    load_entry_point('PyUtilib==5.1.3556', 'console_scripts',\
  \ 'pypi_downloads')()\n  File \"/tmp/tmpego0l9bo/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 489, in load_entry_point\n    return get_distribution(dist).load_entry_point(group,\
  \ name)\n  File \"/tmp/tmpego0l9bo/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 2852, in load_entry_point\n    return ep.load()\n  File \"/tmp/tmpego0l9bo/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 2443, in load\n    return self.resolve()\n  File \"/tmp/tmpego0l9bo/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 2449, in resolve\n    module = __import__(self.module_name, fromlist=['__name__'],\
  \ level=0)\n  File \"/tmp/tmpego0l9bo/lib/python2.7/site-packages/pyutilib/dev/pypi_downloads.py\"\
  , line 21, in <module>\n    import xmlrpc\nImportError: No module named xmlrpc\n"
generated_using:
- --help
